src/h2load.cc

```diff
@@ -1493,6 +1647,7 @@ int main(int argc, char **argv) {
     shared_nva.emplace_back(":authority", config.host);
   }
   shared_nva.emplace_back(":method", config.data_fd == -1 ? "GET" : "POST");
+  shared_nva.emplace_back("user-agent", "h2load nghttp2/" NGHTTP2_VERSION);
```

Comment:
I overlooked explicitly mentioning this change in the initial pull request. Is this acceptable? There could be room to make this more configurable (command-line parameter etc) if you want me to expand it.

Adding the user agent field with a default value has helped in my testing/analysis, especially public-facing servers where various other user agents (also with no UA string) probe the server.